angular-reactive-validation/package-lock.json

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
As noted earlier, you probably ran `npm install` in the sub directory, causing this file to be created.
**suggestion**: Remove the file.

Reply:
Fixed. removed.